dask_expr/io/parquet.py

```diff
@@ -226,10 +227,11 @@ def _meta(self):
             return meta[column]
         return meta
 
-    @property
+    @cached_property
```

Comment:
Even if we decide against the `pickle` call, caching the `_plan` within the object itself is probably a good idea.